packages/@aws-cdk/aws-sqs/lib/queue.ts

```diff
@@ -275,6 +277,26 @@ export class Queue extends QueueRef {
     }
   }
 
+  public grantReceiveMessages(identity?: iam.IPrincipal) {
```

Comment:
In my head this was always called `grantConsume` (since you're consuming from the queue). What do you think? More accurate or too pretentious?

Reply:
Went for Receive to match AWS documentation nomenclature but open to opinion here. It might well be that both names are right and that having both to enable the user to easily find it might be better.

Reply:
"Consume" has the advantage (in my mind) that it's very clear that you'll be removing messages from the queue as well.

Maybe we can squash both discussions by exposing both `grantConsume` (receive+delete+setvisibilitytimeout) and `grantReceive` (just receive).

But I think that will be a compromise to appease the naysayers. I don't see a realistic scenario in which you ever want to give someone just `receiveMessage` permissions. Maybe a snooper, but not an active component.

Reply:
Roger that - I'll change accordingly.